### PR TITLE
Allow using an env var to determine chrome driver binary

### PR DESCRIPTION
**Pull request by Pierstoval**
My problem is that Panthère doesn't work on Alpine (because on Alpine we should either use the driver coming from the repository, or recompile it manually).

For this, it could be nice to tell Alpine users to use either the package or to recompile it instead of using Panthère's provided binary. And in general, to get the latest version, it's better to propose them to download the package in another way, for them to be sure 😅.

For now, an env var is the best solution to use another binary, to me. 

WDYT?

### CODE REVIEW

File: src/ProcessManager/ChromeManager.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
     public function __construct(?string $chromeDriverBinary = null, ?array $arguments = null, array $options = [])
     {
-        $this->process = new Process([$chromeDriverBinary ?? $this->findChromeDriverBinary()], null, null, null, null);
+        $this->process = new Process([$chromeDriverBinary ?: $this->findChromeDriverBinary()], null, null, null, null);
```
Comment (dunglas) [inline src/ProcessManager/ChromeManager.php]: Why this change?

File: src/ProcessManager/ChromeManager.php [hunk 2]
```diff
@@ -66,6 +66,10 @@ public function quit(): void
 
     private function findChromeDriverBinary(): string
     {
+        if (false !== $binary = \getenv('PANTHERE_CHROME_DRIVER_BINARY')) {
```
Comment (dunglas) [inline src/ProcessManager/ChromeManager.php]: we use `$_ENV` in other places (`getenv` isn't thread safe).

File: src/ProcessManager/ChromeManager.php (hunk 1 above)
Comment (Pierstoval) [reply src/ProcessManager/ChromeManager.php]: To avoid `$chromeDriverBinary` be an empty string 😄

File: src/ProcessManager/ChromeManager.php (hunk 2 above)
Comment (Pierstoval) [reply src/ProcessManager/ChromeManager.php]: Fixed

File: src/ProcessManager/ChromeManager.php (hunk 1 above)
Comment (dunglas) [reply src/ProcessManager/ChromeManager.php]: How can it be?

Comment (Pierstoval) [reply src/ProcessManager/ChromeManager.php]: With a simple developer mistake, I know I might be paranoid, but if you think I should revert this, feel free to ask, and I will

Comment (dunglas) [reply src/ProcessManager/ChromeManager.php]: No it's ok! Can you just document this new env var in the README file please?

**Comment by dunglas** [comment]
Thanks @Pierstoval
